### DISCUSSION TITLE
questions were not listing for review in the sourcing url

### DISCUSSION THREAD

**Discussion by bharathravi-dev** [Sunbird-coKreat/Community#60] [General Discussions]
Once contributor  created questions and submitted for the review. so now reviewer logs in and open the project but questions were not listing under that question set.

@pallakartheekreddy @arun1205 @vinukumar-vs

**Comment by arun1205** [comment]
@BharathTarento looking into this.. will let u know

**Comment by Aparna-karri591** [comment]
@BharathTarento If skip review is disabled while creating the Project, First You have to review as default contribution org reviewer and once it is approved there, you can see the Questions in Sourcing reviewer Portal

**Reply by bharathravi-dev** [reply]
@Aparna-karri591  In the default contribution org reviewer the questions are getting for review. but the questions are not getting in sourcing reviewer.

**Reply by Aparna-karri591** [reply]
@BharathTarento Once You are approved the Questions as default contribution org reviewer, Those approved Questions will reflect to sourcing reviewer

**Reply by Aparna-karri591** [reply]
@BharathTarento Its looks like in dev environment there is Question Publish issue, Please contact to @devops  or @inQuiry Team and this is not a coKreat issue.
